### ISSUE TITLE
[LUA] Error on ShowFloatingHelpNotification

### ISSUE THREAD

**Issue by hellyet** [manups4e/ScaleformUI#132]
![image](https://github.com/manups4e/ScaleformUI/assets/74832935/a14c1722-36d6-42d1-bc58-1adc9428bc40)

```lua
  Notifications:ShowFloatingHelpNotification(
      "My test floating notification :3",
      GetEntityCoords(GetPlayerPed(-1)),
      3000
  )
```

Coords is correct, I don't know why it rising an error.

**Comment by hellyet** [comment]
> Problem seems to be on Native itself 🤔

FXServer version is 6440 
I will try to fix it, if it turns out - I will send a pull-request

**Comment by hellyet** [comment]
Yeah, it is. But i don't have any idea about problem reason 😔.

**Comment by hellyet** [comment]
Okey. I think that is cfx api problem.

**Comment by hellyet** [comment]
I found a way to fix it!
```
    Citizen.CreateThread(function()
        while true do
            Citizen.Wait(5)
            Notifications:ShowFloatingHelpNotification('Test Notification', GetEntityCoords(PlayerPedId()), -1)
        end
    end)
```

This function need to be called every frame.

**Comment by hellyet** [comment]
I'm confused...
I want to split it into two functions: ShowFloatingHelpNotification, that need to be called every frame, and ShowFloatingHelpNotificationOverTime, that accept a duration param and don't need to be called every frame.
What do you think?

But there is one thing: these two functions cannot be called at the same time.

**Comment by manups4e** [comment]
i would not add the one with duration as you have seen it seems to be erroring out.. i would default it to use -1 and remove the duration parameter.

**Comment by hellyet** [comment]
Check my PR, how i did it.

**Comment by hellyet** [comment]
sorry, I forgot to remove the debugging code

**Comment by Local9** [comment]
Closed via #133
